pkg/network/server.go

```diff
@@ -345,6 +345,9 @@ func (s *Server) Start() {
 			s.log.Error("skipping NeoFS BlockFetcher", zap.Error(err))
 		}
 	}
+	if s.config.NeoFSStateSyncExtensions {
```

Comment:
> blockfetcher: add GetMPTNodes

It's related not only to this PR, but to all other PRs like https://github.com/nspcc-dev/neo-go/pull/3827. That is `GetMPTNodes`? What is `SessionExpansionEnabled`?  Describe which feature you're implementing in this PR, because in two months I'll forget what does `GetMPTNodes` mean and `git log` becomes non-informative with such name. We have similar P2P command, so this title is confusing.